daemon/daemon.go

```diff
@@ -198,6 +198,10 @@ func (d *Daemon) PolicyEnabled() bool {
 	return d.conf.Opts.IsEnabled(endpoint.OptionPolicy)
 }
 
+func (d *Daemon) GetDebug() bool {
```

Comment:
I would pay more attention to hound if they'd use a puppy as a logo but we should add a comment to the function nevertheless.